StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/PaymentMethodElement/PaymentMethodElement.swift

```diff
@@ -43,10 +65,28 @@ extension ContainerElement {
     }
 }
 
-extension FormElement: PaymentMethodElement {}
-extension SectionElement: PaymentMethodElement {}
+extension FormElement: PaymentMethodElement {
+    func applyDefaults(params: IntentConfirmParams) -> IntentConfirmParams {
+        applyDefaultsHierarchichally(params: params)
+    }
+}
+
+extension SectionElement: PaymentMethodElement {
+    func applyDefaults(params: IntentConfirmParams) -> IntentConfirmParams {
+        applyDefaultsHierarchichally(params: params)
+    }
+}
+
 extension StaticElement: PaymentMethodElement {
+    func applyDefaults(params: IntentConfirmParams) -> IntentConfirmParams {
+        return params
+    }
```

Comment:
Do we need this since it's already defined in the `PaymentMethodElement` extension?

Reply:
Unfortunately yes, because `applyDefaults` is implemented in the `PaymentMethodElement` extension and `ContainerElement` extension, so it becomes ambiguous.
I tried a few ways to make it work without reimplementing but none worked, so that's why I extracted the actual implementation to a helper method so all the extensions could call it.

Reply:
Scratch that, I just realized that this comment is on the `StaticElement` extension, here it _can_ be removed, my previous comment stands for `FormElement` and `SectionElement`